recipe/script.js

```diff
@@ -0,0 +1,42 @@
+function validTextColor(strColor) {
+    const instanceStyle = new Option().style;
+    instanceStyle.color = strColor; 
+    return instanceStyle === strColor;
+}
+
+let backgroundColor = function() {
+    let userColor = prompt("What background color do you want?", 'rgb(164, 247, 222)');
+    if(!validTextColor(userColor)){
+      console.log("This is not a valid input");
+    }
+    return document.body.style.backgroundColor = userColor;
+  }
+  
+  backgroundColor();
+
+  let textColor = function() {
+    let userColor = prompt("What text color do you want?", 'black');
+    if(!validTextColor(userColor)){
+      console.log("This is not a valid input");
```

Comment:
you'll want to write a return after this `console.log` method. otherwise the method would set the `document.body.style.color` to an invalid color